src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/ciagmigration/CiagInductionMigrationService.kt

```diff
@@ -41,5 +48,12 @@ class CiagInductionMigrationService(
       .uri("/ciag/induction/$prisonNumber")
       .retrieve()
       .bodyToMono(CiagInductionResponse::class.java)
+      .onErrorResume(WebClientResponseException::class.java) {
+        if (it.statusCode.value() == 404) {
+          Mono.empty()
+        } else {
+          Mono.error(it)
+        }
```

Comment:
ktlint insists on the curly braces, with separate lines :(

Reply:
I reckon you can do it on a single line without curly braces, but it doesn't matter:
```
if (it.statusCode.value() == 404) Mono.empty() else Mono.error(it)
```